client/state/data-layer/wpcom-http/utils.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns response data from an HTTP request success action if available
+ *
+ * @param {Object} action may contain HTTP response data
+ * @returns {any|null} response data if available
+ */
+export const getData = action => get( action, 'meta.dataLayer.data', null );
+
+/**
+ * Returns error data from an HTTP request failure action if available
+ *
+ * @param {Object} action may contain HTTP response error data
+ * @returns {any|null} error data if available
+ */
+export const getError = action => get( action, 'meta.dataLayer.error', null );
+
+/**
+ * @typedef {Object} ProgressData
```

Comment:
TIL a new JSDoc pattern. I must admit I was thrown off from the lack of a "thing" to which the docblock is attached.